### ISSUE TITLE
to saas or not to saas

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/themeisle-companion#148]
@selul @abaicus At some point we need to decide if we want to require users to get an external api key ( like akismet for e.g) in order to get access to extra modules/templates, this would allow us to communicate/debug in real time, sync templates/addons.

**Comment by selul** [comment]
@ineagu yes, i think this will be a good use case for premium templates, if we would like to have. Also, this will require to move obfx as separate site imho.
